app/build.gradle

```diff
@@ -97,6 +97,7 @@ dependencies {
         exclude module: 'xpp3'
     }
 
+    implementation 'com.android.support:customtabs:25.0.0'
```

Comment:
use latest support library version and manage it using `$rootProject.supportLibraryVersion`